### ISSUE TITLE
Change default project name "root"

### ISSUE THREAD

**Issue by BrianSetz** [rug-ds-lab/sbt-scala-parent#27]
https://github.com/rug-ds-lab/sbt-scala-parent/blob/master/src/main/scala/rugds/sbt/RugDsSbtPlugin.scala#L47

**Comment by lazovik** [comment]
@BrianSetz and what will be a better default name? should we use a parent directory with root being a suffix? e.g., system-core-root? (I would still add 'root' to distinguish from sub-projects which often share the same name)

**Comment by BrianSetz** [comment]
@lazovik Yes, I think including the parent directory would be sufficient. Either that, or force people enter a name e.g. my-project, and automatically add -root (resulting in my-project-root).
